src/video.py

```diff
@@ -5,25 +5,33 @@
 # Created       : Fri, 29 Jan 2016
 # Last Modified : Sun, 31 Jan 2016
 
+# Built-ins
+from sys import exit as die
+from typing import List, Dict, Optional
 
-from sys import exit as Die
+# 3rd Party
 try:
-    import sys
-    import cv2
-    from colordetection import ColorDetector
+    import cv2                                  # type: ignore
```

Comment:
Can `# type: ignore` be removed? If not, please remove leading spaces in between `np_ndarray` and `# type: ignore`